template.tpl

```diff
@@ -957,11 +964,12 @@ const onsuccess = () => {
 
   _amplitude = copyFromWindow(WRAPPER_NAMESPACE);
   if (!_amplitude) return fail('Failed to load the Amplitude namespace');
+  var instanceName = data.instanceName || '$default_instance';
```

Comment:
Minor, `var` -> `const`?

Reply:
i was checking the rest of the code and seems simo intentionally prefers `var` probably to support older browsers?

Reply:
Actually, in this template I do use the ES6 conventions so avoid `var` where possible. In this case `const` does make the most sense. In the wrapper I used `var` because I'm not sure which transpiler you're using (we could move to ES6 there, too).